program/rust/src/accounts/price.rs

```diff
@@ -91,3 +182,10 @@ impl PythAccount for PriceAccount {
     /// Equal to the offset of `comp_` in `PriceAccount`, see the trait comment for more detail
     const INITIAL_SIZE: u32 = PC_PRICE_T_COMP_OFFSET as u32;
 }
+
+#[cfg(test)]
+impl PythAccount for PriceAccountNew {
+    const ACCOUNT_TYPE: u32 = PC_ACCTYPE_PRICE;
+    /// Equal to the offset of `comp_` in `PriceAccount`, see the trait comment for more detail
+    const INITIAL_SIZE: u32 = PC_PRICE_T_COMP_OFFSET as u32;
```

Comment:
is this right? I think you've allocated some bytes *after* the component array now, so the initial size needs to include the entire array + the subsequent values?